addons/skin.estuary/xml/Variables.xml

```diff
@@ -661,4 +661,7 @@
 		<value condition="VideoPlayer.SubtitlesEnabled">[B]$INFO[VideoPlayer.SubtitlesLanguage][/B]</value>
 		<value>[B]$LOCALIZE[1223][/B]</value>
 	</variable>
+	<variable name="PVRInfoParentalRatingCode">
+		<value condition="!String.IsEmpty(ListItem.parentalratingcode)"> [B][COLOR grey]($INFO[ListItem.parentalratingcode])[/COLOR][/B]</value>
```

Comment:
I think the formatting (COLOR, bold) should not be part of the variable. This may hinder using the var from windows where colours/stype/... is different.

Reply:
Actually, I deleted the variable completely.  The variable was only needed so that the parental rating could appear to be appended to the genre string.  Now that the parental rating has been removed from the EPG screen and given its own line plus label in the dialogue screen, I can just refer to it directly via $INFO like any other property.

If another window needs access to the parental rating, like the PVRInfoPanel, then it can also use $INFO with the property name.

Reply:
> Actually, I deleted the variable completely. 

Great. I was about to suggest exactly this.